benchto-service/src/main/java/io/trino/benchto/service/model/Measurement.java

```diff
@@ -109,22 +145,21 @@ public boolean equals(Object o)
             return false;
         }
         Measurement that = (Measurement) o;
-        return Objects.equals(name, that.name);
+        return metric.equals(that.metric);
```

Comment:
Why `unit` does not matter here? `Hibernate` strongly depends on correct equals / hashcode and it looks like two different object could be treated as same, od I miss something?

Reply:
This is why I switched to comparing metrics, to also account for the unit (and attributes). The `name` and `unit` properties that remain here are denormalized from the metric.